app-tasks/rf/src/rf/commands/reprocess_sentinel.py

```diff
@@ -11,26 +11,51 @@
 
 
 @click.command(name='reprocess-sentinel')
-@click.argument('scene_id')
+@click.argument('scene_ids')
 @wrap_rollbar
-def reprocess_sentinel(scene_id):
+def reprocess_sentinel(scene_ids):
     """Fix a sentinel 2 scene by replacing scene metadata bands and Images
 
     Args:
         scene_id (str): Scene id to reprocess
+        scene_ids (str): comma separated list of ids to reprocess
     """
-    logger.info('Starting reprocessing scene: {}'.format(scene_id))
     conn = psycopg2.connect('host={} dbname={} user={} password={}'.format(
         os.environ.get('POSTGRES_HOST', 'postgres'),
         os.environ['POSTGRES_DB'],
         os.environ['POSTGRES_USER'],
         os.environ['POSTGRES_PASSWORD'])
     )
     cur = conn.cursor(cursor_factory=psycopg2.extras.DictCursor)
+
+    ids = [scene.strip() for scene in scene_ids.split(',')]
+
     logger.info('Connected to db')
+
+    for scene_id in ids:
+        logger.info('Starting reprocessing scene: %s', scene_id)
+        try:
+            reprocess_scene_id(cur, conn, scene_id)
+        except Exception as e:
+            conn.rollback()
+            logger.error(
+                'Rolling back transaction. \n' +
```

Comment:
This should be `logger.exception` and can omit including `e` here, the traceback will be included as part of that call.